app/views/landing_page/index.html.erb

```diff
@@ -17,6 +17,8 @@
                             <i class="bi bi-door-open-fill"></i> Log out
                         <% end %>
                     </div>
+
+                    <%= link_to "Profile", profiles_me_path %>
```

Comment:
Suggestion: put this link into line 12, so that clicking on your own email address leads you to your profile

Reply:
Other suggestion: instead of showing just the email, what about showing both the email and the full name? (only showing the full name is probably not a good idea because it is not - yet - guaranteed to be set immediately after signup, which means that the link would be empty and unclickable otherwise)

My idea would be to have something like "Test User (test@example.org)".

Reply:
These are out-of-scope of the user stories. Let's circle back with @Ngo-Tuan-Kiet and get them prioritise for the next sprints.